apps/api/vitest.config.ts

```diff
@@ -4,15 +4,7 @@ import { defineConfig } from "vitest/config";
 export default defineConfig({
   plugins: [tsconfigPaths()],
   test: {
-    reporters: [
-      "verbose",
-      [
-        "json",
-        {
-          outputFile: "./e2e/e2e-server.json",
-        },
-      ],
-    ],
+    reporters: ["verbose", "e2e/utils/log-reporter.ts"],
```

Comment:
Cool if this works, but I thought you need to pass in an instance of your custom reporter.

Reply:
Ah i see the docs say it can be a path.